package.json

```diff
@@ -38,18 +40,24 @@
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/npm": "^10.0.4",
     "@semantic-release/release-notes-generator": "^11.0.4",
+    "@types/node": "^22.9.0",
+    "@types/react": "^18.3.12",
```

Comment:
Same thing - could the types for React match the same major version of React used? I wonder if this is a good opportunity to bump the version of react and react-dom used in the dev deps.